### ISSUE TITLE
multiple labels

### ISSUE THREAD

**Issue by MDLutoronto** [stamen/toner-carto#50]
Labels seem to render every label for every street segment when using in QGIS using Toner. I was looking at Toronto and West Nipissing in Ontario. It does seem to also happen when using maps.stamen.com so doubt it's a QGIS problem.
thanks. 

Marcel Fortin
marcel.fortin@utoronto.ca

**Comment by almccon** [comment]
Agreed, it's definitely not a QGIS problem. For reference, here is the original bug report: https://twitter.com/mapsgis/status/736182877377990656

Location: http://maps.stamen.com/toner/#16/46.3687/-79.9224

<img width="839" alt="screen shot 2016-05-29 at 29 may 12 36 24" src="https://cloud.githubusercontent.com/assets/1212178/15635664/0308ba42-259a-11e6-9713-b781ddefef51.png">

But I note that the default OpenStreetMap map style has the same problem:
http://www.openstreetmap.org/#map=16/46.3682/-79.9233
<img width="851" alt="screen shot 2016-05-29 at 29 may 12 37 18" src="https://cloud.githubusercontent.com/assets/1212178/15635673/2f4077c6-259a-11e6-9cdd-9e199a67e45a.png">

That's because between each intersection these roads are all individual OSM "ways", which is not standard mapping practice. It's due to an old and messy GeoBase import that hasn't been cleaned up. 

See one of the ways here: http://www.openstreetmap.org/way/34739295#map=16/46.3680/-79.9224

I suggest you clean up the area on OSM, by connecting those separate short ways into longer ones. Then Toner will look better once we update the data (although that will probably take a few months before we do it again)
